tools/reactive-controllers/src/RovingTabindex.ts

```diff
@@ -314,37 +79,19 @@ export class RovingTabindexController<T extends HTMLElement>
     manage(): void {
         this.managed = true;
         this.manageTabindexes();
-        this.addEventListeners();
+        super.manage();
     }
 
     unmanage(): void {
         this.managed = false;
         this.updateTabindexes(() => ({ tabIndex: 0 }));
-        this.removeEventListeners();
-    }
-
-    addEventListeners(): void {
-        this.host.addEventListener('focusin', this.handleFocusin);
-    }
-
-    removeEventListeners(): void {
-        this.host.removeEventListener('focusin', this.handleFocusin);
-        this.host.removeEventListener('focusout', this.handleFocusout);
-        this.host.removeEventListener('keydown', this.handleKeydown);
+        super.unmanage();
     }
 
     hostUpdated(): void {
-        if (this.firstUpdated) {
+        if (!this.firstUpdated) {
```

Comment:
Instead of tracking `firstUpdated` here, could we leverage [this.host.hasUpdated](https://lit.dev/docs/components/lifecycle/#hasupdated)?

Reply:
TIL!